packages/error-reporting/README.md

```diff
@@ -0,0 +1,230 @@
+# Node.js module for Stackdriver Error Reporting
```

Comment:
> and either function() {} or () => {}, just be consistent with one of 'em.

With the obvious caveat of -- if you are using arrow functions generally, it is fine to use the `function() {}` style if you actually need it in a particular spot (the syntaxes are not semantically identical).